projects/Scion/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  pkgs,
+  lib,
+  sources,
+} @ args: {
+  packages = {
+    inherit
+      (pkgs)
+      scion
+      scion-apps
+      scion-bootstrapper
+      ioq3-scion
+      pan-bindings
+      ;
+  };
+  nixos = {
+    modules.services.scion = "${sources.inputs.nixpkgs}/nixos/modules/services/networking/scion/scion.nix";
+    tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
```

Comment:
I am not sure if we should link this one here, it isn't render by `overview/default.nix`. [PR](https://github.com/NixOS/nixpkgs/pull/298737/files#diff-ea2b936b45b9e9753598327dd5497b25aedd816a95c8f5593991e4dc26d03505) link with test